tests/tensorflow2/test_keras.py

```diff
@@ -521,8 +522,8 @@ def test_weights_collections(out_dir, tf_eager_mode):
 
     trial = smd.create_trial(path=out_dir)
     # can't save gradients in TF 2.x
-    assert len(trial.tensor_names()) == (5 if is_tf_2_2() and tf_eager_mode else 6)
-    assert len(trial.tensor_names(collection=CollectionKeys.BIASES)) == 0
+    assert len(trial.tensor_names()) == (7 if is_tf_2_2() and tf_eager_mode else 8)
+    assert len(trial.tensor_names(collection=CollectionKeys.BIASES)) == 2
```

Comment:
are biases saved when user selects only weights collection? is this expected behavior?